frontend/app/components/redesign/components/landing/ToolCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Link } from '@remix-run/react'
+import { PillTag } from './PillTag'
+import arrowOutwardIcon from '~/assets/images/landing/arrow-outward.svg'
+
+export type ToolCardProps = {
+  children?: React.ReactNode
+  title: string
+  tags: string[]
+  icon: string
+  to?: string
+  href?: string
+  target?: '_blank' | '_self' | '_parent' | '_top'
+  className?: string
+}
+
+export const ToolCard = ({
+  children,
+  title,
+  tags,
+  icon,
+  to,
+  href,
+  target = '_blank',
```

Comment:
Can maybe keep either of `to` and `href`, and distinguish between how the link behaves based on `target` (`_parent` or `_blank` or default `undefined`)?